superset-frontend/package.json

```diff
@@ -37,7 +37,7 @@
     "src/setup/*"
   ],
   "scripts": {
-    "_lint": "eslint --ignore-path=.eslintignore --ext .js,.jsx,.ts,tsx .",
+    "eslint": "eslint --ignore-path=.eslintignore --ext .js,.jsx,.ts,tsx",
```

Comment:
killing `_lint` with a simple `eslint` command with the parameters that should always be used, but **importantly** removing that `.` at the end that prevents using this construct to run a specific folder of file